src/main/java/uk/gov/hmcts/reform/ccd/util/RestConstants.java

```diff
@@ -1,15 +1,19 @@
 package uk.gov.hmcts.reform.ccd.util;
 
-@SuppressWarnings("ALL")
+@SuppressWarnings({"ALL", "java:S1075"})
```

Comment:
Why "ALL" is not sufficient? :) Though it seems like it is only complaining about S1075, so ALL could be dropped.